.githooks/pre-push

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Run all ci scripts locally before pushing. To enable this hook, run:
+# $ git config core.hooksPath .githooks
+
+BASE_DIR="$(dirname $0)"
+tests="${BASE_DIR}"/../ci_scripts/travis.*.sh
```

Comment:
If we decide to have common repo to store all the test scripts then one additional step to clone that repo will be required here.